examples/morpheusvm/cmd/morpheus-cli/cmd/deploy.go

```diff
@@ -0,0 +1,100 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/ava-labs/avalanchego/tests/fixture/tmpnet"
+	"github.com/spf13/cobra"
+
+	"github.com/ava-labs/hypersdk/examples/morpheusvm/consts"
+	"github.com/ava-labs/hypersdk/examples/morpheusvm/rpc"
+	"github.com/ava-labs/hypersdk/examples/morpheusvm/tests/workload"
+	"github.com/ava-labs/hypersdk/tests/fixture"
+	"github.com/ava-labs/hypersdk/utils"
+)
+
+const (
+	owner      = "morpheus-cli"
+	devNetPort = "9560"
+)
+
+var deployCmd = &cobra.Command{
```

Comment:
Can we instead add this via the root handler, so that it can be shared across VMs https://github.com/ava-labs/hypersdk/blob/87957cdab7a2baa5fcba116a8552c8e30ee56ad4/examples/morpheusvm/cmd/morpheus-cli/cmd/root.go#L65 ?